alignak_backend/app.py

```diff
@@ -1080,6 +1080,40 @@ def pre_host_patch(updates, original):
         del updates['_updated']
 
 
+def after_insert_host(items):
+    """
+    Hook after host inserted.
+
+    :param items: host fields
+    :type items: dict
+    :return: None
+    """
+    for dummy, item in enumerate(items):
+        overall_state = 0
+
+        acknowledged = item['ls_acknowledged']
+        downtimed = item['ls_downtimed']
+        state = item['ls_state']
+        state = state.upper()
+
+        if acknowledged:
+            overall_state = 1
+        elif downtimed:
+            overall_state = 2
+        else:
+            if state == 'UNREACHABLE':
+                overall_state = 3
+            elif state == 'DOWN':
+                overall_state = 4
+
+        # Do not care about services... when inserting an host,
+        # services are not yet existing for this host!
+
+        # Host overall was computed, update the host overall state
+        lookup = {"_id": item['_id']}
+        patch_internal('host', {"_overall_state_id": overall_state}, False, False, **lookup)
```

Comment:
Îndeed, yes, it modifies the _etag. this is why I had to modify the tests ... but how do you modify the returned _etag? Please do the modification ...